### PR TITLE
add: make dev-all target

### PR DESCRIPTION
**Pull request by trombonekenny**
Adds a make target to run a development PL and workspace host.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,8 @@ dev: start-support
 	@yarn dev
 dev-workspace-host: start-support
 	@yarn dev-workspace-host
+dev-all: start-support
+	@$(MAKE) -s -j2 dev dev-workspace-host
```
Comment (jonatanschroeder) [inline Makefile:26]: `-s` is not needed here because `dev` and `dev-workspace-host` commands start with a `@`.

**Review by jonatanschroeder — COMMENTED** [review]
I'll leave it to @mwest1066 or @nwalters512 to approve it, particularly regarding the target name, but this would simplify my work a bit.

Maybe also add a `start-all` target?

**Comment by jonatanschroeder** [comment]
Maybe also as this to the docs?

File: Makefile (hunk 1 above)
Comment (trombonekenny) [reply Makefile:26]: Yeah, I think I added `-s` before I put `@` in the front. Confirmed, it's not needed.

Comment (trombonekenny) [reply Makefile:26]: Ah, it is still useful:

```
bash-5.2# make dev-all
make[1]: Entering directory '/PrairieLearn'
[nodemon] 3.1.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,ts,json,sql
[nodemon] starting `yarn dev:no-watch`
[nodemon] 3.1.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `yarn dev:no-watch`
info: Workspace server listening on port 8081
info: Workspace host ready
info: PrairieLearn server ready, press Control-C to quit
info: Go to https://localhost:443
```

**Comment by echuber2** [comment]
Perhaps it would be good to add some instructions about the new command here, or at least to link to the workspaces article: https://prairielearn.readthedocs.io/en/latest/installingLocal/
This page explains "make dev" and "make start," but it doesn't have anything else to say about workspaces. A lot of people seem to find this set of instructions and not the dedicated workspaces article.

File: docs/installingLocal.md [hunk 2]
```diff
@@ -35,6 +35,9 @@ This page describes the procedure to run PrairieLearn within Docker, but using a
   # Or, run PrairieLearn like it's run in production.
   make start
 
+  # To support workspaces in local development, use `make dev-all` and `make start-all` to run
```
Comment (jonatanschroeder) [inline docs/installingLocal.md:38]:
```suggestion
  # To support workspaces in local development, use `make dev-all` or `make start-all` to run
```

**Review by nwalters512 — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -22,13 +22,17 @@ dev: start-support
...
```
[the remark this answers is not in the document]
Comment (jonatanschroeder) [reply Makefile:26]: https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

[the remark this answers is not in the document]
Comment (nwalters512) [reply Makefile:26]: Looks like that is in fact the way to go!

**Comment by jonatanschroeder** [comment]
Maybe also change? https://github.com/PrairieLearn/PrairieLearn/blob/f01ee5337d76828f57b54aafb839cd5eaa943bcf/docker/init.sh#L13

**Review by jonatanschroeder — APPROVED** [review]

**Pull request merged**